snappi_ixnetwork/protocolmetrics.py

```diff
@@ -0,0 +1,122 @@
+from snappi_ixnetwork.timer import Timer
+
+
+class ProtocolMetrics(object):
+    # Currently paging of the statistic view is not handled
+    # TODO Need to enhance when device groups statistics reach
+    # more than one page.
+
+    _SUPPORTED_PROTOCOLS_ = [
+        "bgpv4"
+    ]
+    _PROTO_NAME_MAP_ = {
+        "bgpv4": 'BGP Peer Per Port',
+        "bgpv4_drill": 'BGP Peer Drill Down'
```

Comment:
please add a TODO here mentioning that the fate of this is not yet decided (just as a reminder for us)

Reply:
the reason why I'm saying this is because we may want a data structure like this:

```python
_PROTO_NAME_MAP_ = {
    "bgpv4": {'per_port': '', 'drill_down': ''}
}
```